assembl/static2/js/app/components/common/termsForm.jsx

```diff
@@ -96,19 +106,18 @@ type OutputProps = {
 export type Props = OutputProps & QueryProps & LegalContentsQuery;
 
 export const mapDataToProps = ({ data }: Object) => {
-  const text = get(data, 'legalContents.termsAndConditions', '');
+  const termsAndConditionsText = get(data, 'legalContents.termsAndConditions', '');
+  const privacyPolicyText = get(data, 'legalContents.privacyPolicy', '');
   return {
     ...data,
-    text: text
+    termsAndConditionsText: termsAndConditionsText,
+    privacyPolicyText: privacyPolicyText
```

Comment:
you will need 2 different `mapDataToProps` and 2 containers to bind `text` value. I would create `TermsForm` and `PrivacyPolicyForm` that render the same dumb component with different `headerTitle`, `text` and `handleAcceptButton` props

Reply:
Yes, exactly! Please don't merge this as is.